tests/dataSources_test.go

```diff
@@ -1333,6 +1431,93 @@ var _ = Describe("DataSources", func() {
 			})
 		})
 
+		Context("with multiple architectures", func() {
+			var cronArchs []architecture.Arch
+
+			BeforeEach(func() {
+				cronArchs = []architecture.Arch{
+					architecture.AMD64,
+					architecture.ARM64,
+					architecture.S390X,
+				}
+				cronTemplate.Annotations[data_sources.DataImportCronArchsAnnotation] = "amd64,arm64,s390x"
```

Comment:
Do we have a test for when the template defines less archs than there are WorkloadArchitectures?

Reply:
Good idea, added.